debezium-server-eventhubs/src/main/java/io/debezium/server/eventhubs/HashFunction.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.server.eventhubs;
+
+/**
+ * Enumeration of supported hash functions for message key hashing.
+ */
+public enum HashFunction {
```

Comment:
Can the calculation be pushed here so istead of having `switch` statement above the `HashFunction` enaum value will be able to calculate hash itself.